theme/templates/resource-landing-page/modals.html

```diff
@@ -1937,15 +1937,26 @@ <h3 class="panel-title">Sharing status</h3>
     </div>
     {#======= Manage access modal  window ends =======#}
 
-    {% block delete-self-access-to-resource %}
-        <div id="dialog-confirm-delete-self-access" title="Warning! Loss of access" style="display:none">
-            <p><span class="ui-icon ui-icon-alert" style="float:left; margin:12px 12px 50px 0;"></span>
-                This action will remove your access to this resource and you will no longer be able to
-                access it without someone else sharing it with you again. This is not reversible,
-                so <strong>Remove</strong> with caution.
-            </p>
-        </div>
-    {% endblock %}
+    <div id="dialog-confirm-delete-self-access" class="access-dialog-title" title="Warning! Loss of access"> 
+        <p><span class="ui-icon ui-icon-alert access-warning"></span>
+            This action will remove your access to this resource and you will no longer be able to
+            access it without someone else sharing it with you again. This is not reversible,
+            so <strong>Remove</strong> with caution.
+        </p>
+    </div>
+    
+    <div id="dialog-confirm-change-share-permission" class="access-dialog-title" title="Warning! Loss of owner access">
```

Comment:
For this warning, it's better to show it without a pop up. For consistency, we try to use this feature as confirmation dialogs. Please use this code instead to show this warning:
```
$("#div-invite-people").find(".label-danger").remove(); // Remove previous alerts
$("#div-invite-people").append("<span class='label label-danger'><strong>Failed to add access: </strong>The selected user already has access.</span>");
```